### DISCUSSION TITLE
About ProductInstance class.

### DISCUSSION THREAD

**Discussion by looklooklook123** [ansys/pyansys-geometry#917] [Q&A] [answered]
code:
from ansys.geometry.core import launch_modeler_with_spaceclaim
modeler = launch_modeler_with_spaceclaim(222,api_version = 22)

The Spaceclaim will run. But wait a minute, the code will wrong, says timeout.
Maybe because this:
in product_instance.py
    _manifest_path_provider(version: int, available_installations: Dict, manifest_path: str = None)

i don't have "Presentation.ApiServerAddIn.Manifest.xml",i had use everything to find it.

**Comment by RobPasMue** [comment, marked as answer]
Hi @looklooklook123 - thanks for opening this discussion!

From what I see in your commands, you are using SpaceClaim 22R2 to run PyAnsys Geometry. PyAnsys Geometry is only compatible with SpaceClaim starting on 23R2 - meaning that you will not be able to communicate with prior versions. Sorry for the inconveniences.

The supported versions can be found here: https://geometry.docs.pyansys.com/version/stable/getting_started/compatibility.html#ref-ansys-comp

**Reply by RobPasMue** [reply]
@ansys/pyansys-geometry-maintainers - if you agree with the answer please "Mark as answer".

**Reply by Eddysca** [reply]
Hi @looklooklook123,

I am experiening the same issue. Did you manage to solve this at the end? If so how did you manage to do this?
Thanks in advance for the help.

Edoardo

**Reply by RobPasMue** [reply]
Hi @Eddysca! Due to your comments on #919 I believe it has to do with the fact that 2023R2 does not have Service Pack 1 installed in your case. Let's try to figure out the solution on that thread =)

**Reply by Eddysca** [reply]
Hi @RobPasMue, thanks for your reply. Yes I can confirm that I am missing the Service Pack 1 installation. Can it be installed on the student version?

**Reply by RobPasMue** [reply]
HI @Eddysca - I think that if you try downloading the latest Ansys Student Version you should have access to the SP1 =) otherwise, Ansys 24R1 is about to be released, maybe you can wait an extra week?
